content/dstu2/care-plan.md

```diff
@@ -0,0 +1,50 @@
+---
+title: CarePlan | DSTU 2 API
+---
+
+# CarePlan
+
+* TOC
+{:toc}
+
+## Terminology Bindings
+
+<%= terminology_table(:care_plan, :dstu2) %>
+
+## Search
+
+Search for CarePlans that meet supplied query parameters:
+
+    GET /CarePlan?:parameters
+
+### Parameters
```

Comment:
Do we need to add an Implementation Notes section to call out the modifier element fields we will not return?  Looks like this could apply to CarePlan.activity.detail.status and CarePlan.activity.detail.prohibited

https://github.com/cerner/fhir.cerner.com/blob/master/content/dstu2/device.md#search